tests/bsim/bluetooth/mesh/src/test_lcd.c

```diff
@@ -86,7 +86,8 @@ static void test_args_parse(int argc, char *argv[])
 	bs_args_parse_all_cmd_line(argc, argv, args_struct);
 }
 
-static const struct bt_mesh_models_metadata_entry *dummy_meta_entry[] = {};
+static const struct bt_mesh_models_metadata_entry dummy_meta_entry[] = {{.len = 0,
```

Comment:
```suggestion
static const struct bt_mesh_models_metadata_entry dummy_meta_entry[];
```
I'm pretty sure this should be sufficient, since this is not a pointer anymore and doesn't need memory allocation.

Reply:
Test fails on my machine if this is not initialized properly.

Reply:
Try **[1]**:
```
static const struct bt_mesh_models_metadata_entry dummy_meta_entry[1];
```